### DISCUSSION TITLE
Error when using this template (need help 😞 )

### DISCUSSION THREAD

**Discussion by ttmottt** [obytes/react-native-template-obytes#324] [Q&A] [answered]
Hello there, 

After many tries, i was not able to install properly this template. 

I had an issue between MMKV & React Native. I tried to upgrade to    ` "react-native-mmkv": "3.0.0-beta.6",`  but now i have this 

```
 ERROR  Error: Failed to create a new MMKV instance: react-native-mmkv 3.x.x requires TurboModules, but the new architecture is not enabled!
* Downgrade to react-native-mmkv 2.x.x if you want to stay on the old architecture.
* Enable the new architecture in your app to use react-native-mmkv 3.x.x.`

```
I tried this also in App.config.ts 

```
 'expo-build-properties',
      {
        ios: {
          newArchEnabled: true,
        },
        android: {
          newArchEnabled: true,
          kotlinVersion: '1.7.22', // this is for softinput package
        },
      },
      
```

Someone is stuck as me ? 

Regards !

**Comment by 0x1337ak** [comment]
@ttmottt did you run prebuild before building

**Reply by ttmottt** [reply]
Yes i follow a this page  
https://docs.expo.dev/guides/new-architecture/

Npx expo prebuild --clean
Npx expo start

**Comment by ttmottt** [comment]
🆕  > I retried the full installation and tested it through the Android Emulator. Good news, it's working! However, I continue to have an issue with Expo Go.

```
 ERROR  Error: Failed to create a new MMKV instance: The native MMKV Module could not be found.
* Make sure react-native-mmkv is correctly autolinked (run `npx react-native config` to verify)
* Make sure gradle is synced.
* Make sure you rebuilt the app., js engine: hermes
```

This error was the sameas below  before i migrated to mmkv 3.x.x

**Reply by 0x1337ak** [reply, marked as answer]
You can't use expo go when you have native code for more information 
https://docs.expo.dev/workflow/using-libraries/#determining-third-party-library-compatibility:~:text=If%20you%20answered%20yes%20to%20any%20of%20these%20questions%2C%20then%20you%20should%20create%20a%20development%20build%20to%20use%20the%20library%20in%20your%20project

**Reply by ttmottt** [reply]
Thanks for your answer !  Good to know  👍
